### PR TITLE
SignUp and SignIn

### LINKED ISSUE #12: SignIn and Signup page

### PR DESCRIPTION
**Pull request by ucheuzor**
User can register or sign up using google sign up.  Also routed link to login page for already registered users.  Form was validated using the Formik library and Yup validationSchema.
signup
SignIn validation page
signin validation

signup2

### CODE REVIEW

File: src/component/SignUp/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+import  './Signup.css';
+import {useFormik} from 'formik';
+import * as Yup from 'yup';
+import img1 from './signup.svg';
+
+
+
+const SignUp = () =>{
+
+      const phoneRegex = RegExp(
```
Comment (teezyfortune) [inline src/component/SignUp/SignUp.js:12]: Well done, why don't you create an helpers function that handles the validation

**Comment by 2besweet** [comment]
This is good for merging

**Review by koladeore — APPROVED** [review]

**Review by teezyfortune — APPROVED** [review]

File: src/component/SignUp/SignUp.js (hunk 1 above)
Comment (ucheuzor) [reply src/component/SignUp/SignUp.js:12]: I also taught of it though. Let me just continue with other part first. I can still revisit that end, maybe create a component that handles the validation separately

**Review by tasiukwaplong — APPROVED** [review]

**Comment by 2besweet** [comment]
This is good but apply all necessary changes

File: src/component/SignUp/SignUp.js (hunk 1 above)
Comment (2besweet) [reply src/component/SignUp/SignUp.js:12]: Pls try to revisit that

**Pull request merged**